### PR TITLE
Update to Ruby 2.0

### PR DESCRIPTION
**Pull request by p0deje**
* Use Ruby 1.9 hash style
* Use `%i()` symbol array shorthand
* Change Watir API usage from (:how, what) to (how: what)

### CODE REVIEW

**Comment by p0deje** [comment]
@titusfortner The diff is pretty huge but it would be great if you could take a look and merge if it's okay.

**Comment by p0deje** [comment]
@titusfortner Do you know when you have a chance to review it?

**Comment by p0deje** [comment]
@titusfortner I'd like to merge this in the next couple of days unless you have any objections.

File: checkbox_spec.rb [hunk 1]
```diff
@@ -11,24 +11,24 @@
...
-      expect(browser.checkbox(:name, /new_user_interests/)).to exist
-      expect(browser.checkbox(:value, "books")).to exist
-      expect(browser.checkbox(:value, /books/)).to exist
+      expect(browser.checkbox(id: "new_user_interests_books")).to exist
+      expect(browser.checkbox(id: /new_user_interests_books/)).to exist
+      expect(browser.checkbox(name: "new_user_interests")).to exist
+      expect(browser.checkbox(name: /new_user_interests/)).to exist
+      expect(browser.checkbox(value: "books")).to exist
+      expect(browser.checkbox(value: /books/)).to exist
       # not sure what :text is supposed to represent here
```
Comment (titusfortner) [inline checkbox_spec.rb]: I haven't run this yet to see the issue, but this should get fixed or removed.

File: browser_spec.rb [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
   describe "#exists?" do
     after do
-      browser.window(:index, 0).use
-      browser.windows[1..-1].each { |win| win.close }
+      browser.window(index: 0).use
+      browser.windows[1..-1].each(&:close)
```
Comment (titusfortner) [inline browser_spec.rb]: Nice use of to_proc :+1:  (does rubocop do this automatically?)

File: checkbox_spec.rb (hunk 1 above)
Comment (p0deje) [inline checkbox_spec.rb]: I think it's better to do that separately from Ruby 2 chages.

Comment (titusfortner) [inline checkbox_spec.rb]: That's fine; just wanted to make note of it as long as I'm going through all of the code. :)

File: browser_spec.rb (hunk 2 above)
Comment (p0deje) [inline browser_spec.rb]: It reports it, but I wasn't correcting it automatically.

File: image_spec.rb [hunk 3]
```diff
@@ -32,99 +32,99 @@
...
   end
 
   # Manipulation methods
   describe "#click" do
     it "raises UnknownObjectException when the image doesn't exist" do
-      expect { browser.image(:id,    'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:class, 'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:src,   'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:alt,   'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
+      expect { browser.image(id:    'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
```
Comment (titusfortner) [inline image_spec.rb]: Everywhere else seems to have single space after colon, instead of matching the spacing.

File: browser_spec.rb [hunk 4]
```diff
@@ -104,40 +104,40 @@
...
+      browser.goto(WatirSpec.url_for("frames.html", needs_server: true))
+      title = browser.element(tag_name: 'title').text
       browser.frame.body.exists? # switches to frame
       expect(browser.title).to eq "Frames"
     end
   end
 
   describe ".start" do
-    not_compliant_on(:webdriver, :safariwatir) {
+    not_compliant_on(webdriver: :safariwatir) {
```
Comment (titusfortner) [inline browser_spec.rb]: This one got copied wrong

**Comment by titusfortner** [comment]
Whew, finished. Sorry it took so long. Other than the 2 above I don't see any other issues.

File: image_spec.rb (hunk 3 above)
Comment (p0deje) [inline image_spec.rb]: Agree, I'll remove that.

File: browser_spec.rb (hunk 4 above)
Comment (p0deje) [inline browser_spec.rb]: Good catch!

**Comment by p0deje** [comment]
@titusfortner Thanks, fixed the 2 issues and merge after specs are green.

**Pull request merged**